jfr-container-action/setup.sh

```diff
@@ -1,17 +1,11 @@
 #!/usr/bin/env bash
 set -e
 
-echo "Set up Jenkins war package."
-jar -cvf /app/jenkins/jenkins.war /app/jenkins/*
-
-echo "Download plugins."
-java -jar /app/bin/jenkins-plugin-manager.jar --war /app/jenkins/jenkins.war --plugin-file "$3" --plugin-download-directory=/usr/share/jenkins/ref/plugins
-
-if [ $# == 4 ]
+if [ $# == 3 ]
 then
     echo "Set up JCasC."
-    cp "$4" ${CASC_JENKINS_CONFIG}
+    cp "$3" ${CASC_JENKINS_CONFIG}
 fi
```

Comment:
You should exit if `$# < 2` or if `$# >= 4` to avoid unexpected surprises.

Reply:
You are right. But based on the definition of `action.yml`, it won't make the number of parameters `$# < 2` or if `$# >= 4`. In order to make it error free, I'll add this condition.